vllm/entrypoints/llm.py

```diff
@@ -107,13 +109,16 @@ class LLM:
         hf_overrides: If a dictionary, contains arguments to be forwarded to the
             HuggingFace config. If a callable, it is called to update the
             HuggingFace config.
+        compilation_config: Either an integer or a dictionary. If it is an integer,
+            it is used as the level of compilation optimization. If it is a dictionary,
+            it can specify the full compilation configuration.
         **kwargs: Arguments for :class:`~vllm.EngineArgs`. (See
             :ref:`engine_args`)
 
     Note:
         This class is intended to be used for offline inference. For online
         serving, use the :class:`~vllm.AsyncLLMEngine` class instead.
-    """
+    """ # noqa
```

Comment:
nit: is it because of the line-too-long error? If so, can you please fix L113?

Reply:
this is a triple quote, adding `# noqa` can ignore the line length limit for all lines inside the triple quote.

Reply:
Yes please don't add `noqa` in this case. Please adhere to the line limit as much as possible.